tests/IntegrationTests/BlobUnitTests.fs

```diff
@@ -74,11 +73,17 @@ let blobMainTests =
             value |> shouldEqual "thing")
 
         testCase "Cloud Blob Client relates to the same data as the type provider" (fun _ ->
-            Expect.contains (Local.Containers.CloudBlobClient.ListContainers() |> Seq.map(fun c -> c.Name)) "samples" "")
+            Expect.contains (
+                Local.Containers.CloudBlobClient.ListContainersSegmentedAsync(null).Result
```

Comment:
Instead of using .Result - please use task {}

Reply:
We have to do that for the tp itself as well

Reply:
The net standard branch is hacked together at the moment - there are loads of .`Result` scattered around currently that aren't in master. This is because there are breaking changes between the .NET Standard and .NET Framework version of the Azure SDK at the *same* version - literally switching between the two breaks your code.

Just to get things working on .NET Core, I'm happy for .Results to be there at the minute, but before result we'll clean them all up.